.circleci/config.yml

```diff
@@ -219,6 +222,42 @@ jobs:
       - store_artifacts:
           path: integration_tests/screenshots
 
+  accessibility_test:
+    executor: integration-tests
+    steps:
+      - browser-tools/install-chrome
```

Comment:
do we definitely need these? The existing integration_test job doesn't install them

Reply:
I switched everything to run on chrome (integration and accessibility) as electron was masking some of the a11y issues.  Can change the integration tests back to electron if you like though?

Reply:
ah! I see sorry - good shout